includes/environment.inc

```diff
@@ -323,6 +323,11 @@ function drush_locate_root($start_path = NULL) {
     if ($follow_symlinks && is_link($path)) {
       $path = realpath($path);
     }
+    // Check to see if we have any bootstrap classes in this location.
+    $bootstrap_class_dir = $path . '/drush/bootstrap';
+    if (is_dir($bootstrap_class_dir)) {
+      _drush_add_commandfiles(array($bootstrap_class_dir), DRUSH_BOOTSTRAP_DRUSH);
```

Comment:
1. The better constant is DRUSH_BOOTSTRAP_NONE.
1. Now this function does two things. Would be better to move this commandfile search elsewhere. I don't know where offhand.